daml-lf/interpreter/src/main/scala/com/digitalasset/daml/lf/speedy/Compiler.scala

```diff
@@ -72,13 +72,13 @@ private[lf] object Compiler {
   )
 
   object Config {
-    val Default = Config(
+    val Default: Config = Config(
```

Comment:
I love that we have types on all the function returns.
But for these `val` types, I don't care so much. You could argue its a bit overkill to say `Config` twice.

Reply:
I usually skip the return type for case like that (when the right hand side is a fully applied constructor of a case class).
I do not have strong feeling though.

Reply:
After discussion, I think I'll remove these `Config` type annotations - the RHS expression looks suitably stable and, as rightly pointed out, it is not adding any real value here that can be discerned.